Source/SteveSloth/Public/MyEnemyDeadState.h

```diff
@@ -0,0 +1,39 @@
+/****************************************************************************************
+* Copyright: SteveSloth
+ * Name:Guy Lapensee
+ * Script: MyTemplate.h
+ * Date:May 3, 2024
+ * Description:Enemy Dead State
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "MyEnemyBaseState.h"
+#include "MyEnemyBaseClass.h"
+#include "MyEnemyDeadState.generated.h"
+
+/**
+ * 
+ */
+UCLASS()
+class STEVESLOTH_API UMyEnemyDeadState : public UMyEnemyBaseState
+{
+	GENERATED_BODY()
+
+public:
+	AMyEnemyBaseClass* Enemy;
```

Comment:
Also, this is just getting a reference to the script, you need to get a reference to the actual enemy